pyke/targetpixelfile.py

```diff
@@ -66,14 +70,18 @@ def __init__(self, path, aperture_mask=None,
 
     def _quality_mask(self, quality_bitmask):
         """Returns a boolean mask which flags all good-quality cadences.
-
         Parameters
         ----------
         quality_bitmask : int
             Bitmask. See ref. [1], table 2-3.
         """
         return (self.hdu[1].data['QUALITY'] & quality_bitmask) == 0
 
+    def header(self, ext=0):
+        """Returns the header for a given extension."""
+        return self.hdu[ext].header
+
+
```

Comment:
This repeats the next three lines of code

Reply:
👍 👍 👍